packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Server/EnvVars/index.tsx

```diff
@@ -19,7 +25,7 @@ const EnvironmentVariablesComponent = ({ signals: { editor }, store }) => {
     editor.deleteEnvironmentVariable({ name });
   };
 
-  const envVars = store.editor.currentSandbox.environmentVariables;
+  const envVars: any = currentSandbox.environmentVariables;
```

Comment:
Why would this become an `any`?

Reply:
`environmentVariables` is an Array so this was giving me type errors since in the code below is trying to use `toJson` method and it doesn’t exits in an array type
Anyway that is fixed now with the changes @christianalfoni added :)